services/order.py

```diff
@@ -0,0 +1,37 @@
+from datetime import datetime
+
+from django.db import transaction
+
+from db.models import Order, Ticket, User, MovieSession
+
+
+@transaction.atomic
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: datetime = None
+) -> Order:
+    user = User.objects.get(username=username)
+    with transaction.atomic():
+        order = Order.objects.create(user=user)
+        if date:
+            order.created_at = datetime.strptime(
+                date, "%Y-%m-%d %H:%M"
+            )
+            order.save()
+        for ticket in tickets:
+            Ticket.objects.create(
+                row=ticket["row"],
+                seat=ticket["seat"],
+                movie_session=MovieSession.objects.get(
+                    id=ticket["movie_session"]
+                ),
+                order=order,
+            )
+    return order
+
+
+def get_orders(username: str = None) -> Order:
```

Comment:
Check your type annotations in the code, you return not a single `Order` object, but a `QuerySet` of `Order` objects. Change your type annotation to encounter that.